bin/processGsRawData.sh

```diff
@@ -53,7 +53,8 @@ function sanityChecking() {
 	#
 	local _batch="${1}"
 	local _controlFileBase="${2}"
-	local _controlFileBaseForFunction="${_controlFileBase}.${FUNCNAME[0]}"
+	local _dataType="${3}"
```

Comment:
is dit script nu niet overbodig? zit deze code ook niet in PullAndProcessGsRawData.sh?

Reply:
ja klopt hoor.. dit script ga ik ook weggooien nadat alles af is getest